drafts/str4d-stratum/draft1.rst

```diff
@@ -0,0 +1,256 @@
+Abstract
+========
+
+This ZIP describes the Zcash variant of the Stratum protocol, used by miners to
+communicate with mining pool servers.
+
+
```

Comment:
Add this boilerplate:

``This document follows the common conventions defined for ZIPs in [ZIP-1]_."